### PR TITLE
Add required documentation badges.

### PR DESCRIPTION
**Pull request by dlecocq**
@b4hand This one's kind of an oddball. The team is really help team -- I just helped one of the helpsters make this happen. I don't actually know if this is something that's being used at this point :-/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 Zendesk Search
 ==============
+![Status: Production](https://img.shields.io/badge/status-production-green.svg?style=flat)
+![Team: Big Data](https://img.shields.io/badge/team-big_data-green.svg?style=flat)
+![Scope: Internal](https://img.shields.io/badge/scope-internal-green.svg?style=flat)
+![Open Source: No](https://img.shields.io/badge/open_source-:--%28-green.svg?style=flat)
```
Comment (b4hand) [inline README.md]: This repo appears to be public not internal.

**Comment by dlecocq** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (dlecocq) [inline README.md]: LGTM

**Comment by b4hand** [comment]
LGTM

**Comment by dlecocq** [comment]
LGTM

**Pull request merged**